node-bb/README.md

```diff
@@ -0,0 +1,175 @@
+# TripleA Forums
+
+The software that powers the TripleA Forums is [NodeBB](https://github.com/NodeBB/NodeBB). 
+This document is instructions on how to build and run the Docker container of the forums using an 
+[official Docker container of NodeBB](https://hub.docker.com/r/nodebb/docker) as the base image.
+
+## Building
+
+1. Make a directory on your machine for where the config file and upload folder will live. ```mkdir -p /opt/triple-forums```
+
+2. Make a directory for uploaded files. Inside the upload directory, create folders category, emoji, files, profile, 
+sounds, and system.
+   ```shell script
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 3 spaces](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6342386)